tests/cli/test_deploy_commands.py

```diff
@@ -201,6 +204,38 @@ def test_manual_proxy_retargeting(testerchain, click_runner, token_economics):
     assert proxy_deployer.target_contract.address == untargeted_deployment.address
 
 
+def test_batch_deposits(click_runner,
+                        testerchain,
+                        agency_local_registry, # FIXME: remove
+                        mock_allocation_infile,
+                        token_economics):
+
+    #
+    # Main
+    #
+
+    deploy_command = ('allocations',
+                      '--registry-infile', agency_local_registry.filepath,
+                      '--allocation-infile', mock_allocation_infile,
+                      # '--allocations-outfile', MOCK_ALLOCATION_REGISTRY_FILEPATH,
```

Comment:
Are we keeping this line (while commented) for some reason?

Reply:
✅